resources/views/editnote.blade.php

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
+
+<head>
+    <title>Login V19</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!--===============================================================================================-->
+    <link rel="icon" type="image/png" href="images/icons/favicon.ico" />
+    <!--===============================================================================================-->
+
+    <link rel="stylesheet" type="text/css"
+        href="http://localhost:8081/LARAVEL/laravel_quicktask/resources/css/util.css">
+    <link rel="stylesheet" type="text/css"
+        href="http://localhost:8081/LARAVEL/laravel_quicktask/resources/css/main.css">
+    <!--===============================================================================================-->
+</head>
+
+<body>
+
+    <div class="limiter">
+        <div class="container-login100">
+            <div class="wrap-login100 p-l-55 p-r-55 p-t-65 p-b-50">
+                <form class="login100-form validate-form" method="POST" action="{{ route('editNote') }}">
+                    @csrf
+                    <span class="login100-form-title p-b-33">
+
+                        EDIT NOTE
```

Comment:
```suggestion
                        {{  trans('notes.edit') }}
```
Localization, xóa dòng thừa nha